data_structures/binary_tree/binary_tree_traversals.py

```diff
@@ -0,0 +1,190 @@
+class Node:
+    """
+    A Node has data variable and pointers
+    to its left and right nodes.
+    """
+
+    def __init__(self, data):
+        self.left = None
+        self.right = None
+        self.data = data
+
+
+def make_tree() -> Node:
+    root = Node(1)
+    root.left = Node(2)
+    root.right = Node(3)
+    root.left.left = Node(4)
+    root.left.right = Node(5)
+    return root
+
+
+def preorder(root):
+    """
+    PreOrder traversal: visit root node
+    then its left subtree followed
+    by right subtree.
+    preorder(make_tree())

```

Comment:
```suggestion

    >>> preorder(make_tree())
```
Those three >>> (plus a space that follows) make it a doctest.  https://docs.python.org/3/library/doctest.html

Reply:
ok